src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -28,7 +32,10 @@ public ListCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
         }
 
-        return new ListCommand();
+        if (StateManager.getState() != null) {
```

Comment:
Should use the atMainPage()/atCasePage() check here rather than checking the element